src/ApiGateways/Caller/MASA.PM.Caller/Callers/EnvironmentCaller.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MASA.PM.Caller.Callers
+{
+    public class EnvironmentCaller : HttpClientCallerBase
+    {
+        private readonly string _prefix = "/api/v1/env";
+
+        public EnvironmentCaller(IServiceProvider serviceProvider) : base(serviceProvider)
+        {
+            Name = nameof(EnvironmentCaller);
+        }
+
+        protected override string BaseAddress { get; set; } = "http://localhost:6030";
+
+        //protected override string AppId { get; set; } = "masa-pm-service-admin";
```

Comment:
Why not use AppId?

Reply:
this version uses HttpClientCaller. and next version will update caller and use AppId